### DISCUSSION TITLE
월드 내에서의 의상 교체

### DISCUSSION THREAD

**Discussion by jefflim69** [naverz/zepeto-studio-kor#1026] [5. Q&A]
제작한 월드 내에서  자체  제작한 의상을 Player 에게  교체 적용 해 보고자 합니다 .
(피팅룸에서 옷을 갈아 입듯이 , 현재의 월드에서만 적용하여 스크린 샷 적용이 목적) .

Q 1. Zepeto 정책에 위반 되는 내용인지 알려 주시면 감사하겠습니다 .

Q 2. 허용 된다면 구현을 어떤 방식으로 접근하면 좋을지 부탁 드립니다 .

(Unity 개발 환경입니다 )

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 입어보기 기능 제공 준비 중에 있으며 정확한 오픈 시기는 아직 말씀드릴 수 없는 점 양해 부탁 드립니다.
이 기능이 제공된다면 자체 제작한 의상을 Player가 착용하는 것은 정책적으로 가능합니다 :) 
감사합니다.
